metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -2110,8 +2135,9 @@ def get_internal_workunits(self) -> Iterable[MetadataWorkUnit]:  # noqa: C901
                                 explore_reachable_views.add(view_name.include)
                     except Exception as e:
                         self.reporter.report_warning(
-                            f"{model}.explores",
-                            f"failed to process {explore_dict} due to {e}. Run with --debug for full stacktrace",
+                            title="Failed to process explores",
+                            message="Failed to process explore dictionary. Run with --debug for full stacktrace",
+                            context=f"Explore Details: {explore_dict}, Exception: {e}",
```

Comment:
`exc=e`. also we probably don't need to say "run with --debug ..."

Reply:
Say goodbye to that